Events/Ch28.event

```diff
@@ -0,0 +1,247 @@
+BLST [18,22] Ballista
+END_MAIN
+
+ALIGN 4
+
+BeginningScene:
+
+LOAD1 0x1 MetalGear
+ENUN
+Text(PlaceholderText)
+LOAD1 0x1 FinalFantasy
+ENUN
+Text(PlaceholderText)
+LOAD1 0x1 Castlevania1
+ENUN
+LOAD1 0x1 SimonsQuest
+ENUN
+Text(PlaceholderText)
+LOAD1 0x1 Units
+ENUN
+Text(PlaceholderText)
+//LOAD1 0x1 FillerGuy
+ENDB
+
+EndingScene:
+Text(PlaceholderText)
+CHECK_EXISTS KojimaNPC
+BEQ  0x1 0xC 0x0
+	TurnNPC(KojimaNPC)
+	TurnNPC(SakaguchiNPC)
+	TurnNPC(IgaNPC)
+GOTO 0x2
+LABEL 0x1
+	GOTO 0x2
+LABEL 0x2
+MNCH 29
+ENDA
+
+// Events
+EventAfterExitingPrepScreen:
+Text(PlaceholderText)
+ENDA
+
+SilverAxeVillage:
+Text(PlaceholderText)
+SVAL 0x3 SilverAxe
+GIVEITEMTO 0xFFFF
+ENDA
+
+ChannelingXavier:
+Text(PlaceholderText)
+LOAD1 0x01 XavierPeople
+REMA
+ENDA
+
+KojimaGet:
+Text(PlaceholderText)
+TurnAlly(Kojima)
+TurnNPC(KojimaNPC)
+TurnEnemy(SakaguchiNPC)
+TurnEnemy(IgaNPC)
+ENDA
+
+SakaguchiGet:
+Text(PlaceholderText)
+TurnAlly(Sakaguchi)
+TurnEnemy(KojimaNPC)
+TurnNPC(SakaguchiNPC)
+TurnEnemy(IgaNPC)
+ENDA
+
+IgaGet:
+Text(PlaceholderText)
+TurnAlly(Iga)
+TurnEnemy(KojimaNPC)
+TurnEnemy(SakaguchiNPC)
+TurnNPC(IgaNPC)
+ENDA
+
+RaiseBridge:
+Text(RaisingBridge)
+TILECHANGE 6
+ENDA
+
+
+// Manual Movement
+
+// Scripted Fights
+
+// Units
+MetalGear:
+UNIT Kojima Assassin Kojima Level(20, Enemy, 0) [18,9] 0x00 0x00 0x0 0x00 [KillingEdge, PoisonSword] GuardTileAI
+UNIT 0xB6 Berserker Kojima Level(20, Enemy, 1) [23,10] 0x00 0x00 0x0 0x00 [PoisonAxe] ChaseOnceApproachedAI 
+UNIT 0xB6 Berserker Kojima Level(20, Enemy, 1) [11,0] 0x00 0x00 0x0 0x00 [PoisonAxe] ChaseOnceApproachedAI 
+UNIT 0xB6 WyvernLord Kojima Level(20, Enemy, 1) [12,8] 0x00 0x00 0x0 0x00 [PoisonSword] ChaseOnceApproachedAI 
+UNIT 0xB6 WyvernLord Kojima Level(20, Enemy, 1) [9,11] 0x00 0x00 0x0 0x00 [PoisonSword] ChaseOnceApproachedAI 
+UNIT 0xB6 WyvernLord Kojima Level(20, Enemy, 1) [15,11] 0x00 0x00 0x0 0x00 [PoisonSword] ChaseOnceApproachedAI 
+UNIT 0xB6 Sniper Kojima Level(20, Enemy, 1) [17,7] 0x00 0x00 0x0 0x00 [PoisonBow] AttackInRangeAI //ballistician
+UNIT 0xB6 Sniper Kojima Level(20, Enemy, 1) [18,8] 0x00 0x00 0x0 0x00 [PoisonBow] AttackInRangeAI //ballistician
+UNIT 0xB6 Rogue Kojima Level(20, Enemy, 1) [18,3] 0x00 0x00 0x0 0x00 [PoisonSword] [0x10, 0x00, 0x00, 0x00] //StealingThiefAI minus the 0x05
+UNIT 0xB6 Rogue Kojima Level(20, Enemy, 1) [25,5] 0x00 0x00 0x0 0x00 [PoisonSword] [0x10, 0x00, 0x00, 0x00] 
+UNIT 0xB6 Rogue Kojima Level(20, Enemy, 1) [29,10] 0x00 0x00 0x0 0x00 [PoisonSword] [0x10, 0x00, 0x00, 0x00] 
+UNIT Inafune Thief Kojima Level(9, Enemy, 1) [0,5] 0x00 0x00 0x0 0x00 [BodyRing, GoldGem, GoldGem, GoldGem] LootingThiefAI
+UNIT
+
+FinalFantasy:
+UNIT Sakaguchi Sage Sakaguchi Level(20, Enemy, 0) [6,16] 0x00 0x00 0x0 0x00 [Forblaze] GuardTileAI
+UNIT 0xB7 Sage Sakaguchi Level(10, Enemy, 1) [6,14] 0x00 0x00 0x0 0x00 [Fimbulvetr] AttackInRangeAI 
+//UNIT 0xB7 Sage Sakaguchi Level(10, Enemy, 1) [8,19] 0x00 0x00 0x0 0x00 [Bolting] GuardTileAI 
```

Comment:
Why are these units commented out? IF they're not being used, please remove them from the event file. If they're being removed temporarily (for debugging or something), please use an `ifdef`/`ifndef` guard (on `__DEBUG__` probably)